_posts/2019-05-31-ucx-dgx.md

```diff
@@ -57,22 +58,24 @@ today) that figures out which networking system to use.
      width="100%" /></a>
 
 Python users today don't have much access to these network libraries, except
-through MPI, which is sometimes not ideal.
+through MPI, which is sometimes not ideal. ([Try searching for "infiniband" on
+PyPI.](https://pypi.org/search/?q=infiniband))
 
 This led us to create [UCX-Py](https://github.com/Akshay-Venkatesh/ucx-py/)
```

Comment:
ucx-py url can be changed to https://github.com/rapidsai/ucx-py though the redirect will also work

Reply:
Thanks!  Resolved.